tests/basic.spec.js

```diff
@@ -1,8 +1,28 @@
 import OasisApp from "index.js";
+import { serializePathv1 } from "../src/helperV1";
+
+const context = "oasis-core/consensus: tx for chain testing";
 
 test("check address conversion", async () => {
   const pkStr = "17483e0883cf71e2fe4e12f42d1448d06f4274a73b9b6f560c5ed01a32745276";
   const pk = Buffer.from(pkStr, "hex");
   const addr = OasisApp.getBech32FromPK(pk);
   expect(addr).toEqual("oasis1zayruzyreac79ljwzt6z69zg6ph5ya988wdk74svtmgp5vn52fmqg7uz69");
 });
+
+test("check prepare chunks function", async () => {
```

Comment:
Now that we are reworking this, it would be good to check that things work well for:
- an empty context
- we reject a context that is 256 bytes